packages/lesswrong/components/users/ReviewVotingPage.tsx

```diff
@@ -611,6 +652,47 @@ const QuadraticVotingButtons = withStyles(quadraticVotingButtonStyles, {name: "Q
   </div>
 })
 
+const reactionsButtonStyles = (theme: ThemeType) => ({
+  root: {
+    ...theme.typography.commentStyle,
+    padding: 8,
+    borderRadius: 8,
+    backgroundColor: 'white',
+    marginRight: 4,
+    marginLeft: 4,
+    marginBottom: 4,
+    boxShadow: '0px 1px 5px 0px rgba(0, 0, 0, 0.2)',
+    cursor: 'pointer',
+    '&:hover': {
+      backgroundColor: 'rgba(240,240,240,1)'
+    }
+  },
+  active: {
+    color: 'white',
+    backgroundColor: 'rgba(50, 50, 50, 1)',
+    '&:hover': {
+      backgroundColor: 'rgba(100,100,100,1)'
+    }
+  }
+})
+
+
+const ReactionsButton = withStyles(reactionsButtonStyles, {name: "ReactionsButton"})(({classes, postId, vote, votes, reaction }: {classes: ClassesType, postId: string, vote: any, votes: vote[], reaction: string}) => {
```

Comment:
Please don't do this (bypassing `registerComponent` and applying `withStyles` directly). The reason we got diverging, hard-to-merge changes on this file is because I cleaned that up in my branch.

Reply:
This comes with a significant performance penalty, and will probably be forbidden-outright in the near future.